src/ipc/windows/cryptIpc.c

```diff
@@ -0,0 +1,141 @@
+#include "cryptIpc.h"
+#include "ipc.h"
+#include "utils/crypt/crypt.h"
+#include "utils/crypt/ipcCryptUtils.h"
+#include "utils/json.h"
+#include "utils/logger.h"
+#include "utils/memory.h"
+#include "utils/memzero.h"
+#include "utils/oidc_error.h"
+#include "utils/stringUtils.h"
+
+#include <sodium.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <winsock2.h>
```

Comment:
In general, this cryptIpc is similar to its Unix version, but uses Windows Sockets instead of file descriptors